modules/apps/29-fee/keeper/keeper.go

```diff
@@ -53,17 +54,17 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 	return ctx.Logger().With("module", "x/"+host.ModuleName+"-"+types.ModuleName)
 }
 
+// ChanCloseInit wraps the channel keeper's function in order to expose it to underlying app.
+func (k Keeper) ChanCloseInit(ctx sdk.Context, portID, channelID string, chanCap *capabilitytypes.Capability) error {
```

Comment:
This function seems odd, since it won't call the callback. I think we should remove this, and force underlying app to send the message through the message server.

That will ensure that everything gets called correctly. cc: @colin-axner 

Kind of irrelevant to this PR so not blocking on this